index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>Fabricio La Torre</title>
+</head>
+<body>
+    <header class="main-header">
+        <div class="container">
+            <h1 class="mh-logo">
+              <a href="#">La Torre</a>
+                
+            </h1>
+            <!--
+            <nav class="main-nav">
+               
+                <ul class="main-nav-list">
+                  
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a href="#"><img alt="maillogo" src="images/mail.svg"></a>
+                    </li>
+                </ul>
```

Comment:
- [ ] Please remove this commented code from your HTML. :+1: Because commented code makes your code not look clean :cry: